frontend2/src/components/elements/SelectMenu.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { Fragment, useMemo, useState } from "react";
```

Comment:
"Check warning on line 1 in frontend2/src/components/elements/SelectMenu.tsx

GitHub Actions
/ Frontend linting / unit tests

'useState' is defined but never used"

GHA said that ^ woahhh
but yeah, drop it for good snytax?